multiversx_sdk_core/transaction_builders/__init__.py

```diff
@@ -20,5 +22,5 @@
     "DefaultTransactionBuildersConfiguration",
     "ContractCallBuilder", "ContractDeploymentBuilder", "ContractUpgradeBuilder",
     "EGLDTransferBuilder", "ESDTNFTTransferBuilder", "ESDTTransferBuilder", "MultiESDTNFTTransferBuilder",
-    "ESDTIssueBuilder", "RelayedTransactionV1Builder", "RelayedTransactionV2Builder"
+    "ESDTIssueBuilder", "RelayedTransactionV1Builder", "RelayedTransactionV2Builder", "DelegationFactory"
```

Comment:
When `feat/factories` is ready, we should have them in a different folder. E.g. `transaction_factories`. You can also move them in this PR if possible.

Reply:
Moved in `transaction_factories`